pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -31,18 +36,27 @@ import (
 )
 
 var log = logf.Log.WithName("controller_wildflyserver")
-var jbossHome = os.Getenv("JBOSS_HOME")
-var standaloneServerDataDirPath = jbossHome + "/standalone/data"
 
 const (
-	httpApplicationPort int32 = 8080
-	httpManagementPort  int32 = 9990
+	controllerName                      = "wildflyserver-controller"
+	httpApplicationPort           int32 = 8080
+	httpManagementPort            int32 = 9990
+	defaultRecoveryPort           int32 = 4712
+	wildflyServerFinalizer              = "finalizer.wildfly.org"
+	wildflyHome                         = "/opt/jboss/wildfly"
```

Comment:
Please revert this change, in #55, I fixed the server location so that ` jbossHome` points to the correct server location and `standaloneServerDataDirPath` is correct.